docs/server/first_project.rst

```diff
@@ -0,0 +1,59 @@
+.. _first_project:
+
+Your First Project
+==================
+
+Congratulations to your excellent choice of Pootle to manage your translations.
+We hope you'll enjoy your swift transcendance into i10n with the Pootle crew.
+
+Now that you have the server set up, it's time to create some messages to go with
+your very special software. Most likely you have already a set of default messages
+which you want to translate into a second language. This guide will help you get
+started with these translations.
```

Comment:
I suggest "get started with Pootle setup for getting translations for your texts."